PathsActivity.py

```diff
@@ -304,12 +304,12 @@ def _list_tubes_reply_cb(self, tubes):
 
     def _list_tubes_error_cb(self, e):
         """ Log errors. """
-        print('Error: ListTubes() failed: %s', e)
+        print(('Error: ListTubes() failed: %s', e))
 
     def _new_tube_cb(self, id, initiator, type, service, params, state):
         """ Create a new tube. """
-        print('New tube: ID=%d initator=%d type=%d service=%s params=%r \
-state=%d' % (id, initiator, type, service, params, state))
+        print(('New tube: ID=%d initator=%d type=%d service=%s params=%r \
+state=%d' % (id, initiator, type, service, params, state)))
```

Comment:
You don't need the extra parentheses here too, you can either leave it the way it is or use f strings.